app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionDetailViewModel.java

```diff
@@ -88,12 +88,12 @@ private void setSession(@NonNull Session session) {
         this.languageResId = session.lang != null ? decideLanguageResId(session.lang.toUpperCase()) : R.string.lang_en;
     }
 
-    public Maybe<Session> findSession(int sessionId) {
+    public Completable findSession(int sessionId) {
```

Comment:
For this purpose, this method should be `existsSession` else.
A method whose name-prefix is`find` should return the associated thingy.